dpe/src/dpe_instance.rs

```diff
@@ -173,11 +181,11 @@ impl<C: Crypto, P: Platform> DpeInstance<'_, C, P> {
 
     pub(crate) fn generate_new_handle(
         &self,
-        crypto: &mut C,
+        env: &mut impl DpeEnv,
```

Comment:
As a general comment in this file, should we just take `&mut impl Crypto` in some of these because it doesn't need platform? Its kind a tradeoff though, It will probably make the callers a bit uglier. What do you think?

Reply:
I liked always passing DpeEnv for consistency. Espcially because it makes it easier if we later add stuff to platform since everyone has access to it already as part of the Env.

Reply:
I'm good with that